packages/styles/scss/themes/expressive/components/_select.scss

```diff
@@ -26,4 +26,12 @@
     height: rem(20px);
     width: rem(20px);
   }
+  .#{$prefix}--select--inline.#{$prefix}--select--invalid .#{$prefix}--label,
+  .#{$prefix}--select--inline.#{$prefix}--select--invalid
+    .#{$prefix}--form__helper-text {
+    margin-top: 1rem;
+  }
+  .#{$prefix}--form__helper-text {
+    line-height: 1rem;
```

Comment:
can we combine these two under the same selector? or is it different?

Reply:
they're different, i tried combining already

Reply:
ok, I think the helper text originally supposed to be 1rem line-height but because carbon hardcoded it we didn't touch the type-face of it. I guess we can just merge it in as it is now